benchmarks/benchmarks/cutest/dfovec.py

```diff
@@ -312,7 +312,7 @@ def dfovec(m, n, x, nprob):
             tmp2 = np.exp(-x[5] * (temp - x[8]) ** 2)
             tmp3 = np.exp(-x[6] * (temp - x[9]) ** 2)
             tmp4 = np.exp(-x[7] * (temp - x[10]) ** 2)
-            fvec[i] = y5[i] - (x[0] * tmp1 + x[1] * tmp2 + x[2] * tmp3 + x[3] * tmp4)  # noqa
+            fvec[i] = y5[i] - (x[0] * tmp1 + x[1] * tmp2 + x[2] * tmp3 + x[3] * tmp4)
```

Comment:
Do you know what this was for?

Reply:
no

Reply:
actually, probably PEP8 line length. It's <88 though so can be removed.